data.sql

```diff
@@ -1,5 +1,12 @@
 /* Populate database with sample data. */
+INSERT INTO animals (id,name, date_of_birth , neutered, escape_attempts, weight_kg) 
+VALUES (1,'Agumon', '2020-02-03', 0, TRUE, 10.23);
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (id,name, date_of_birth , neutered, escape_attempts, weight_kg) 
+VALUES (2,'Gabumon', '2018-11-15', 2, TRUE, 8);
+
+INSERT INTO animals (id,name, date_of_birth , neutered, escape_attempts, weight_kg) 
+VALUES (3,'Pikachu', '2021-01-07', 1, FALSE, 15.04);
+
+INSERT INTO animals (id,name, date_of_birth , neutered, escape_attempts, weight_kg) 
+VALUES (4,'Devimon', '2017-05-12', 5, TRUE, 11);
```

Comment:
- [ ] Kindly make sure that all the values are matched with the data. As you can see for escape_attempts you have displayed the value True which is not right. 
You have shuffled the values of neutered and escape_attempts. i.e neutered should be a boolean value and escape_attempts a numeric value.
```sql
INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
   VALUES ('Agumon', '2020/2/3', '0', 'true', '10.23');
```

Reply:
Great suggestion, thanks. DONE